scripts/make_preloaded_export.py

```diff
@@ -1,51 +1,67 @@
 #!/usr/bin/env python
-"""
-Make the butler export yaml file to be used by ap_verify runs.
+# This file is part of ap_verify_ci_dc2.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Make the butler export yaml file to be used by ap_verify runs.
+
+This script must be run after **any** change to the preloaded repository;
+otherwise, ingestion may fail or the changes may not be visible.
 """
 
-import argparse
 import logging
 import os
 import sys
 
 import lsst.log
 import lsst.skymap
 import lsst.daf.butler as daf_butler
-import lsst.ap.verify as ap_verify
```

Comment:
Right, I forgot that the old version introduced an unnecessary dependency on `ap_verify` CLI. Thanks for removing it!